### PR TITLE
Tab tinting issue fix + navigation button colors set programmatically + version update

### LINKED ISSUE #132: Set colors programmatically
Similar to `stepperLayout.setBackgroundColor(color)` there should be a method `stepperLayout.setButtonColor(color)`  or `stepperLayout.setTextColor(color)` to set the text color of a button color programmatically.

Although the above would be enough for my use case, as an alternative `StepViewModel` could have to two more fields `backButtonTextColor` and `nextButtonTextColor` to set the text color indivially for each step.

### PR DESCRIPTION
**Pull request by zawadz88**
This PR resolves issue #207 i.e. fixes tab circle background color tinting so that it is possible now to use colors with transparency.
It also resolves #132 i.e. adds an option to set bottom navigation colors programmatically.

It also updates the library version to 4.3.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
...
 
+## [4.3.0]
+### Added
+- A way to set bottom navigation button colors programmatically via:
+ `StepperLayout#setNextButtonColor(int)`/`StepperLayout#setNextButtonColor(ColorStateList)`,
+ `StepperLayout#setCompleteButtonColor(int)`/`StepperLayout#setCompleteButtonColor(ColorStateList)` and
+ `StepperLayout#setBackButtonColor(int)`/`StepperLayout#setBackButtonColor(ColorStateList)`
+
+### Fixed
+- Tab circle background color so that it is possible now to use colors with transparency.
```
Comment (RobertZagorski) [inline CHANGELOG.md]: Maybe adding an issue reference numer here might be good?

File: sample/src/androidTest/java/com/stepstone/stepper/sample/SetButtonColorProgrammaticallyActivityTest.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import static com.stepstone.stepper.test.StepperNavigationActions.clickNext;
+import static org.hamcrest.Matchers.allOf;
+
+/**
+ * Performs tests on a stepper on which we change the button colors in code.
+ *
+ * @author Piotr Zawadzki
+ */
+@LargeTest
+public class SetButtonColorProgrammaticallyActivityTest extends AbstractActivityTest<SetButtonColorProgrammaticallyActivity> {
```
Comment (RobertZagorski) [inline sample/src/androidTest/java/com/stepstone/stepper/sample/SetButtonColorProgrammaticallyActivityTest.java]: What do you think about actually testing tinting with custom Matcher in Espesso tests?

Comment (zawadz88) [reply sample/src/androidTest/java/com/stepstone/stepper/sample/SetButtonColorProgrammaticallyActivityTest.java]: Good idea, I'll see what can be done :)

Comment (zawadz88) [reply sample/src/androidTest/java/com/stepstone/stepper/sample/SetButtonColorProgrammaticallyActivityTest.java]: Done :)

**Pull request merged**